src/App.js

```diff
@@ -20,80 +20,98 @@ class App extends Component {
       mouseOverPlace: '',
     }
 
-    this.handleSubmit = this.handleSubmit.bind(this)
-    this.handleChange = this.handleChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleLocationChange = this.handleLocationChange.bind(this);
+    this.handleLoad = this.handleLoad.bind(this);
```

Comment:
You want to avoid arrow due to performance. I also had to look up the occasions of .bind() since i wasn't confident on when to use them. But it looks like (always) when referencing {this.___}

Good resource
https://medium.com/shoutem/react-to-bind-or-not-to-bind-7bf58327e22a